docs/build/program/apis/data-client.md

```diff
@@ -1076,3 +1065,11 @@ await data_client.remove_binary_data_from_dataset_by_ids(
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Find part ID
+
+To copy the ID of your machine part, select the part status dropdown to the right of your machine's location and name on the top of its page and select the copy icon next to **Part ID**.
```

Comment:
I think select makes sense when you choose an option from a dropdown. This looks like a button (am on mobile so can't check easily)
```suggestion
To copy the ID of your machine part, select the part status dropdown to the right of your machine's location and name on the top of its page and click the copy icon next to **Part ID**.
```